tests/core/util/test_streamable.py

```diff
@@ -580,11 +602,11 @@ class DataclassMissing(Streamable):
             pass
 
 
-def test_streamable_inheritance_missing():
+def test_streamable_inheritance_missing() -> None:
 
     with raises(DefinitionError):
 
         @streamable
         @dataclass(frozen=True)
-        class StreamableInheritanceMissing:
+        class StreamableInheritanceMissing:  # type: ignore[type-var] # we want to test invalid here, hence the ignore.
```

Comment:
Comment above the class?

Reply:
See [21df10b](https://github.com/Chia-Network/chia-blockchain/pull/10539/commits/21df10bf03a1db07bb8eaf9a8860a362369f87d6)